package.json

```diff
@@ -79,14 +79,18 @@
     "package.json": "npx sort-package-json"
   },
   "dependencies": {
-    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#feature/PP-278/PP-329/upgrade_contracts"
+    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#PP-642/test-utils-server",
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#feature/PP-278/PP-329/upgrade_contracts",
+    "@rsksmart/rif-relay-server": "github:rsksmart/rif-relay-server#PP-642/test-utils-server",
+    "config": "^3.3.8"
```

Comment:
should I move this to devDependencies

Reply:
That is what confuse me. I understand that config needs to be available in a production, but this repo is for testing purposes, is there ever going to be used in production?

Reply:
updated